src/components/UserInfo/UserInfo.jsx

```diff
@@ -1 +1,15 @@
-export const UserInfo = () => <>Put the user here</>;
+import './UserInfo.scss';
+
+export const UserInfo = ({ user }) => {
+  const { name, username, email } = user;
+
+  return (
+    <p>
+      {`Posted by ${username}`}
```

Comment:
In the initial markup and tests, there were no usernames, so I think this is redundant.
<img width="648" alt="nicknames" src="https://github.com/mate-academy/react_static-list-of-posts-js/assets/139244901/c523cf2d-da71-43a5-9d30-fe90b8c45650">
The appearance also hints at it.